js/src/Figure.js

```diff
@@ -741,7 +741,25 @@ var FigureView = widgets.DOMWidgetView.extend({
         var initialTraces = _.cloneDeep(this.model.get("_data"));
         var initialLayout = _.cloneDeep(this.model.get("_layout"));
 
-        Plotly.newPlot(that.el, initialTraces, initialLayout).then(
+        let holder = document.createElement('div');
+        holder.classList.add('js-plotly-plot-inner');
```

Comment:
Is the `'js-plotly-plot-inner'` class associated with anything else? Or just a class for future use?

Reply:
Yeah this is what the plotly object binds to, in case you want to add any additional controls or anything to the container (like bqplot and Altair have some buttons)